packages/astro/src/core/constants.ts

```diff
@@ -45,6 +45,11 @@ export const DEFAULT_404_COMPONENT = 'astro-default-404.astro';
  */
 export const DEFAULT_500_COMPONENT = 'astro-default-500.astro';
 
+/**
+ * A response with one of these status codes will create a redirect response.
+ */
+export const REDIRECT_STATUS_CODES = [301, 302, 303, 307, 308, 300, 304] as const;
```

Comment:
Maybe worth reusing somewhere else? like for `redirect` 2nd argument?

Reply:
Oh maybe! I didn't look at other usages. I'll check that